features/FFF_CFE_Integration_V4/NP-full-1-V5-2.feature

```diff
@@ -32,6 +32,21 @@ Feature:
             | 2021-05-10   | rent              | id7       | 10.00  |
             | 2021-04-10   | rent              | id8       | 10.00  |
             | 2021-03-10   | rent              | id9       | 10.00  |
+        And I add the following outgoing details for "child_care" in the current assessment:
+            | payment_date | housing_cost_type | client_id | amount |
+            | 2021-05-10   |                   | id7       | 10.00  |
+            | 2021-04-10   |                   | id8       | 10.00  |
+            | 2021-03-10   |                   | id9       | 10.00  |
+        And I add the following outgoing details for "maintenance_out" in the current assessment:
+            | payment_date | housing_cost_type | client_id | amount |
+            | 2021-05-10   |                   | id7       | 10.00  |
+            | 2021-04-10   |                   | id8       | 10.00  |
+            | 2021-03-10   |                   | id9       | 10.00  |
+        And I add the following outgoing details for "legal_aid" in the current assessment:
+            | payment_date | housing_cost_type | client_id | amount |
+            | 2021-05-10   |                   | id7       | 10.00  |
+            | 2021-04-10   |                   | id8       | 10.00  |
+            | 2021-03-10   |                   | id9       | 10.00  |
```

Comment:
The other thing of note is please don't change tests like this one - it's a copy (hopefully) of a spreadsheet test, and so shouldn't be in scope for any changes